src/libs/DistanceRequestUtils.ts

```diff
@@ -67,7 +67,7 @@ function getMileageRates(policy: OnyxInputOrEntry<Policy>, includeDisabledRates
  * @returns [currency] - The currency associated with the rate.
  * @returns [unit] - The unit of measurement for the distance.
  */
-function getDefaultMileageRate(policy: OnyxInputOrEntry<Policy>): MileageRate | undefined {
+function getDefaultMileageRate(policy: OnyxInputOrEntry<Policy>, localeCompare?: (a: string, b: string) => number): MileageRate | undefined {
```

Comment:
Can we make `localeCompare` required instead of optional? That should avoid anyone forgetting the param.